Sources/TuistCore/Cache/Cache.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public struct Cache: Equatable, Hashable {
+    public struct Flavor: Equatable, Hashable {
+        public let name: String
+        public let configuration: String
+
+        public init(name: String, configuration: String) {
+            self.name = name
+            self.configuration = configuration
+        }
+    }
+
+    public let flavors: [Flavor]
+
+    public init(flavors: [Flavor]) {
+        self.flavors = flavors
+    }
+
+    public static let `default` = Cache(flavors: [Flavor(name: "development", configuration: "Debug")])
```

Comment:
There's a caveat in this approach. It's possible that the `Debug` configuration doesn't exist. This is very unlikely, but I was wondering of deciding the default should be something that we do later in the process.